packages/template-tab-navigation-ng/src/app.android.scss

```diff
@@ -1,11 +1,4 @@
-// Import app variables
-@import 'app-variables';
-
-// Import the theme’s main ruleset - both index and platform specific.
-@import '~nativescript-theme-core/scss/index';
-@import '~nativescript-theme-core/scss/platforms/index.android';
-
 // Import common styles
-@import 'app-common';
+@import "app-common";
```

Comment:
Remove these files and add an app.scss. :slightly_smiling_face:

Reply:
This is something that was decided about the templates some time ago - we wanted to show at least in one place how to have a common file + branching platform specific files. Basically, boilerplate.